app/src/main/kotlin/io/intrepid/skotlinton/base/BaseFragment.kt

```diff
@@ -17,7 +17,7 @@ import timber.log.Timber
 abstract class BaseFragment<P : BaseContract.Presenter> : Fragment(), BaseContract.View {
 
     protected val skotlintonApplication: SkotlintonApplication
-        get() = activity.application as SkotlintonApplication
+        get() = activity?.application as SkotlintonApplication
```

Comment:
might be out of scope for the for this PR, but we might want to create a method that either returns a nonnull activity or crash. I feel like using the ? operator if activity is null is simply just gonna delay the crash/bug and make it hard to track down

Reply:
! would work too, but I guess people just refrain from using it and just blinding use ?

Reply:
Yeah I'm in-progress on some other style cleanup so I can add that to my list. Though in this particular instance we'll crash on the `as` check anyway